CMakeLists.txt

```diff
@@ -162,7 +167,22 @@ if(BUILD_TEST)
       opencv_highgui
   )
 
-  add_executable(uvc_test src/test.c)
+  if(OpenCVPkg_FOUND)
+    # Our example written in C language but found OpenCV modules does
+    # not always contains C include files. So we need to additional check.
+    # highgui_c.h is currently broken.
+    find_file(highgui_path
+      HINTS /usr/include/opencv4 /usr/local/include/opencv4
```

Comment:
this assumes that OpenCV is installed under `/usr` or `/usr/local`.
that is a bad assumption, since OpenCV may be located elsewhere (e.g. when built manually).
Best to rely on something like `OpenCV_INCLUDE_DIRS` here.

Reply:
Agreed